src/main.py

```diff
@@ -300,6 +299,7 @@ def run(self):
                                     sub.sticky()
                                     print "Submission stickied..."
                             break
+                        else: time.sleep(120)

```

Comment:
I also meant to mention that it's inside the loop from line 247. The loop on line 250 is only for exception handling when the game thread edit fails.